### PR TITLE
Add labels to dataregistry

### PR DESCRIPTION
**Pull request by saulshanabrook**
Screen Shot 2019-08-20 at 4 24 13 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ For a development install, clone the repository and then run the following in th
 ```bash
 pip install .
 jlpm build:dev
+jupyter labextension install .
+jupyter serverextension enable jupyterlab_hdf
```
Comment (telamonian) [inline README.md]: Are you sure that you needed those two lines to make the dev install work? 

- `jupyter labextension link .` runs automatically as part of the `jlpm build:dev` command
- you shouldn't need to manually enable the server extension, as that's handled automatically in `setup.py` (by installing the `jupyter-config/jupyter_notebook_config.d/jupyterlab_hdf.json` appropriately)

If the `build:dev` command and the automatic server extension enabling in `setup.py` aren't working as intended, then that's a separate bug I need to fix.

File: package.json [hunk 2]
```diff
@@ -63,8 +57,7 @@
     "rxjs": "^6.5.2"
   },
   "devDependencies": {
-    "husky": "^3.0.1",
-    "lint-staged": "^9.2.0",
```
Comment (telamonian) [inline package.json]: Why'd you strip out the lint hook stuff?

**Comment by telamonian** [comment]
Thanks @saulshanabrook! The fixes/additions to the typescript files look great.

I did have a couple of questions about the other changes you made, tho

File: package.json (hunk 2 above)
Comment (saulshanabrook) [reply package.json]: Ah, sorry this was an accident! It doesn't work for me locally, so I had to remove it to get anything to build (my weird `execa` issue...) I will add it back.

Comment (telamonian) [reply package.json]: Yeah, that's what I assumed after you mentioned the trouble yesterday

File: README.md (hunk 1 above)
Comment (saulshanabrook) [reply README.md]: That's true, OK I reverted this as well.

**Comment by telamonian** [comment]
LGTM. I'll merge it in now

**Pull request merged**